### PR TITLE
Migrate `sci-m` to builtins as `sci-embodied`

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Migrates `sci-m` from `if-plugins` to `builtins` and rename to `sci-embodied`.
Addresses https://github.com/Green-Software-Foundation/if/issues/683

### CODE REVIEW

File: src/builtins/sci-embodied/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
+```
+
+You can run this example `manifest` by executing the following command from the project root:
+
+```sh
+npm i -g @grnsft/if
+npm i -g @grnsft/if-plugins
+ie --manifest ./examples/manifests/test/sci-embodied.yml --output ./examples/outputs/sci-embodied.yml
```
Comment (manushak) [inline src/builtins/sci-embodied/README.md:108]: the manifest path is changed, please, update

File: src/builtins/sci-embodied/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+name: sci-embodied
+description: simple demo invoking sci-embodied
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci-embodied:
+      method: SciEmbodied
+      path: '@grnsft/if-plugins'
```
Comment (manushak) [inline src/builtins/sci-embodied/README.md:87]: the path should be `builtin`

File: src/builtins/sci-embodied/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+## Implementation
+
+IF implements the plugin based on the logic described above. To run the plugin, you must first create an instance of `SciEmbodied`. Then, you can call `execute()` to return `m`.
+
+## Usage
+
+The following snippet demonstrates how to call the `sci-embodied` plugin from Typescript.
+
+```typescript
+import {SciEmbodied} from '@grnsft/if-plugins';
```
Comment (manushak) [inline src/builtins/sci-embodied/README.md:59]: not imported from `@grnsft/if-plugins` anymore

File: src/builtins/sci-embodied/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        resources-total: 8
+      inputs:
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
+```
+
+You can run this example `manifest` by executing the following command from the project root:
+
+```sh
+npm i -g @grnsft/if
```
Comment (manushak) [inline src/builtins/sci-embodied/README.md:106]: remove step `npm i -g @grnsft/if-plugins`

File: src/builtins/sci-embodied/README.md (hunk 3 above)
Comment (jmcook1186) [reply src/builtins/sci-embodied/README.md:59]:
```suggestion
import {SciEmbodied} from 'builtins';
```

File: src/builtins/sci-embodied/README.md (hunk 2 above)
Comment (jmcook1186) [reply src/builtins/sci-embodied/README.md:87]:
```suggestion
      path: 'builtins'
```

File: src/builtins/sci-embodied/README.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+npm i -g @grnsft/if-plugins
```
Comment (jmcook1186) [inline src/builtins/sci-embodied/README.md:107]:
```suggestion
```

File: src/builtins/sci-embodied/README.md (hunk 1 above)
Comment (jmcook1186) [reply src/builtins/sci-embodied/README.md:108]:
```suggestion
ie --manifest manifests/plugins/sci-embodied.yml --output manifests/outputs/sci-embodied.yml
```

**Review by manushak — APPROVED** [review]

File: src/builtins/sci-embodied/README.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+name: sci-embodied
+description: simple demo invoking sci-embodied
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci-embodied:
+      method: SciEmbodied
+      path: 'builtin'
```
Comment (jmcook1186) [inline src/builtins/sci-embodied/README.md:87]:
```suggestion
      path: 'builtins'
```

File: src/builtins/sci-embodied/README.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+## Implementation
+
+IF implements the plugin based on the logic described above. To run the plugin, you must first create an instance of `SciEmbodied`. Then, you can call `execute()` to return `m`.
+
+## Usage
+
+The following snippet demonstrates how to call the `sci-embodied` plugin from Typescript.
+
+```typescript
+import {SciEmbodied} from 'builtin';
```
Comment (jmcook1186) [inline src/builtins/sci-embodied/README.md:59]:
```suggestion
import {SciEmbodied} from 'builtins';
```

**Review by narekhovhannisyan — APPROVED** [review]

**Pull request merged**